src/firejail/x11.c

```diff
@@ -870,6 +862,15 @@ void x11_start_xpra(int argc, char **argv) {
 
 	/* sleep(1); // adding a delay in order to let the server start */
 
+	// wait for server to end
+	while (1) {
```

Comment:
What kind of perf impact will this have, won't it just peg a core at 100%? Can you add a `nanosleep(250)`?